data_prep/introspector.py

```diff
@@ -343,10 +359,19 @@ def populate_benchmarks_using_introspector(project: str, language: str,
     logging.error('No unreached functions found')
     return []
 
-  filenames = [
-      os.path.basename(function['function_filename']) for function in functions
-  ]
-  result = project_src.search_source(project, filenames)
+  if language == 'jvm':
+    filenames = [
+        "%s.java" %
```

Comment:
Can we just do a string concatenation (or use f-string) here?

Reply:
Changed.